packages/flutter/lib/src/rendering/binding.dart

```diff
@@ -40,6 +40,12 @@ mixin RendererBinding on BindingBase, ServicesBinding, SchedulerBinding, Semanti
     _handleSemanticsEnabledChanged();
     assert(renderView != null);
     addPersistentFrameCallback(_handlePersistentFrameCallback);
+    _mouseTracker = MouseTracker(pointerRouter, (Offset offset) {
```

Comment:
create it in a `createMouseTracker` method so that you can override this

Reply:
Done.